tests/v2_tests/test_local.py

```diff
@@ -291,6 +293,45 @@ def test_from_url_create_option(self):
         from_url(path, create=True)
         assert os.path.exists(path) and os.path.isdir(path)
 
+    def test_ppe_profiling(self):
```

Comment:
Please don't extend the `TestLocal(unittest.TestCase)` class. We're slowly moving on from the `unitest` style, to a more pytest-like style where each test is defined as a single and independent function (sometimes with fixtures or parameterized). Please refer to `test_s3.py`.

Reply:
Changed to not use `TestLocal(unittest.TestCase)`.

Currently, `foo.txt` is generated in the directory where pytest runs, do you want to use a different path? (e.g. `/tmp/...`)